### PR TITLE
Service failure page

### PR DESCRIPTION
**Pull request by prakash-gds**
This service failure page will be modified once the actual design page is available

Also, added comments on SEND_EMAIL flag

### CODE REVIEW

File: request_a_govuk_domain/request/views.py [hunk 1]
```diff
@@ -396,17 +401,37 @@ def send_confirmation_email(request) -> None:
...
+            # Once we get NOTIFY_API_KEY from AWS, then this flag should be removed
+            if get_env_variable("SEND_EMAIL", "False") == "True":
+                send_confirmation_email(request)
+        except Exception as e:
+            registration_data = request.session.get("registration_data", {})
+            logger.error(
+                f"""Exception at Application submitted stage. Exception: {type(e).__name__} - {str(e)} ,
+                         Registration data: {registration_data}"""
+            )
+            return HttpResponseRedirect(self.failure_url)
```
Comment (maxf) [inline request_a_govuk_domain/request/views.py:423]: I don't think this will return a 500 error page. It just redirects to a normal 200 page.

**Review by maxf — CHANGES_REQUESTED** [review]
This isn't the right approach if you want to be compliant to HTTP (we do). The error page should also return a 500 status code, just like 404 errors. The way to do this in django is to provide a template for the 500 page, which should be automatically shown in case an exception happens in the code.
Having a test for this would also be better.

**Review by maxf — CHANGES_REQUESTED** [review]
Could you call your template `500.html`. It's an widespread convention and I want to add templates for 404 and 403 errors next.

**Comment by prakash-gds** [comment]
Closing this as the changes are now under https://github.com/co-cddo/domains-register-a-govuk-domain/pull/117

**Pull request closed without merging**